cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -421,8 +421,8 @@ type HostPathMount struct {
 type Patches struct {
 	// Directory is a path to a directory that contains files named "target[suffix][+patchtype].extension".
 	// For example, "kube-apiserver0+merge.yaml" or just "etcd.json". "target" can be one of
-	// "kube-apiserver", "kube-controller-manager", "kube-scheduler", "etcd". "patchtype" can be one
-	// of "strategic" "merge" or "json" and they match the patch formats supported by kubectl.
+	// "kube-apiserver", "kube-controller-manager", "kube-scheduler", "etcd", "kubeletconfiguration".
```

Comment:
Is there a reason that we don't use `kubelet-configuration`?

Reply:
tried to keep the name consistent with the original "kind" KubeletConfiguration -> kubeletconfiguration.